R/SVG.R

```diff
@@ -9,6 +9,8 @@
 #' @param pointsize default point size.
 #' @param standalone Produce a standalone svg file? If \code{FALSE}, omits
 #'   xml header and default namespace.
+#' @param overwrite_page Overwrite the current page when a new page is created.
```

Comment:
This name doesn't accurately reflect what the argument does. And I think it's worth considering whether an argument is needed at all, or `svglite()` should always produce a full svg